src/http/routes/tus/index.ts

```diff
@@ -112,7 +139,7 @@ function createTusServer(
     respectForwardedHeaders: true,
     allowedHeaders: ['Authorization', 'X-Upsert', 'Upload-Expires', 'ApiKey', 'x-signature'],
     maxSize: async (rawReq, uploadId) => {
-      const req = rawReq as MultiPartRequest
+      const req = rawReq.node?.req as MultiPartRequest
```

Comment:
Maybe we should throw here if `rawReq.node` is undefined otherwise it'll just crash on the very next line trying to access `upload` on undefined `req`.